ProcessLib/HeatTransportBHE/BHE/ThermoMechanicalFlowProperties.h

```diff
@@ -27,6 +28,25 @@ struct ThermoMechanicalFlowProperties
     double nusselt_number;
 };
 
+struct AdvectiveThermalResistanceCoaxial
+{
+    double Inner_pipe;
+    double a_annulus;
+    double b_annulus;
+};
+
+struct PipeWallThermalResistanceCoaxial
+{
+    double const Inner_pipe;
+    double const Annulus;
+};
+
+struct GroutAndGroutSoilExchangeThermalResistanceCoaxial
+{
+    double const conductive_b;
+    double const gs;
```

Comment:
what is gs?

Reply:
It is the thermal resistance due to grout-soil exchange. Name has been changed.